docs/gen_doc_stubs.py

```diff
@@ -11,6 +11,8 @@
     if path.name.startswith("_"):
         continue
 
+    # TODO: Not sure of the reference need for this file? But refactor changed this file up a bit,
+    #  so some examination is warranted to see what this might break.
     # base.py doesn't need API docs, main.py is too small to be useful
     if path.name in ["base.py", "main.py"]:
         continue
```

Comment:
The `gen_doc_stubs.py` file is used by `mkdocs` to automatically build API documentation for our docs site. This code block excludes the python classes/functions defined in any `base.py` or `main.py` file from being included in those docs. With the refactor, we now have a base *folder*, so this code doesn't do what it was intended to do anymore:

[Before](https://tsdat.readthedocs.io/en/v0.7.7/API/io/converters/):
<img width="366" alt="image" src="https://github.com/tsdat/tsdat/assets/24307537/eef3d2d9-9a5f-44ac-9926-4e9fc285ebf1">

[After - this PR](https://tsdat--183.org.readthedocs.build/en/183/API/io/base/archive_reader/):
<img width="341" alt="image" src="https://github.com/tsdat/tsdat/assets/24307537/ab433d69-9581-4204-9fb2-4f2c56777172">

I think I prefer the new way. The sections are collapsable because they are submodules, which makes it easier to navigate.


Can you please modify this to be:

```python
if path.name in ["main.py"]:
    continue
```

Reply:
Updated in [f5bd71d](https://github.com/tsdat/tsdat/pull/183/commits/f5bd71d9662b0a3181a71d5436f18a9381ddf5e5).